core/src/main/resources/error/error-classes.json

```diff
@@ -668,6 +668,23 @@
       }
     }
   },
+  "LOCATION_ALREADY_EXISTS" : {
+    "message" : [
+      "Cannot create the location <location> because it already exists."
```

Comment:
do we really need sub error classes here? I think we can do
```
Cannot name the managed table as <identifier>, as its associated location <location> already exists. Please pick a different table name, or remove the existing location first.
```

Reply:
In create table, user can specify the location via a create option explicitly but the rename command doesn't allow this.

Reply:
@cloud-fan One more thing, by saying this `... a different table name ...`, you linked the location issue to (managed) table issue but there are other issues w/ location that are not linked to (managed) table. And this error class can be re-used in other cases too (try to search the word `location` in `error-classes.json`)

Reply:
> user can specify the location via a create option explicitly

I don't think that's a solution. External table is very different from managed table and changing the table type is likely unacceptable to fix this error.

> there are other issues w/ location that are not linked to (managed) table

Can you give some examples? I searched but can't find any.

Reply:
> Can you give some examples?

@cloud-fan For example, this one https://github.com/apache/spark/blob/93d13afe7f0319104e88faa79a8ae72bc6e9215f/core/src/main/resources/error/error-classes.json#L2790-L2794

Reply:
that's for a specific legacy behavior and will go away eventually. Can we give it a dedicated error class?